HTML5-frontend/script.js

```diff
@@ -8,7 +8,6 @@ $(function(){
     setTimeout(initializeTest, 1000);
```

Comment:
Can we at this point change the script ("script.js") filename to something that is more understandable?